packages/discord-error/src/discord-error.ts

```diff
@@ -26,10 +26,20 @@ export type APIError =
 	| { [k: string]: APIError }
 	| string;
 
+/**
+ * Differentiates a Discord error from an OAuth error
+ * @param error - the body text of the error to check
+ * @returns whether the error is a Discord error
+ */
 export function isDiscordError(error: ErrorBody): error is DiscordErrorBody {
 	return "code" in error;
 }
 
+/**
+ * Determines whether the value is a group of errors
+ * @param error The record or piece of the record returned by the Discord API to check
```

Comment:
Inconsistent documentation, with or without the - after the `@param error`

Reply:
There’s a eslint plugin that helps with that `eslint-plugin-tsdoc` or just use `eslint-config-neon` which with you never need to think about such things ever again 😉

Reply:
Ooooh I will check these out, thank you for the tip!

Reply:
Gonna check out neon in another branch, it requires a couple more changes than just the base one. If I tried to add that here ckohen would behead me